src/catalog/src/remote/client.rs

```diff
@@ -33,6 +34,10 @@ pub struct MetaKvBackend {
 /// comparing to `Accessor`'s list and get method.
 #[async_trait::async_trait]
 impl KvBackend for MetaKvBackend {
+    fn as_any(&self) -> &(dyn Any + Send + Sync) {
+        self
+    }
+
```

Comment:
The implementation order differs from trait member definition.

Reply:
🤣